README.md

```diff
@@ -14,149 +19,64 @@ If you want to install a specific version, run the following command with the re
 curl -L https://raw.githubusercontent.com/software-mansion/cairo-coverage/main/scripts/install.sh | sh -s -- v0.1.0
 ```
 
-## Example
+## External tools integration
 
-Let's say you have a following program
+`cairo-coverage` is tool-agnostic which means that it accepts input from any tool. However, these tools need to generate trace data in a specific expected format -
+the same format which is accepted by the [cairo-profiler](https://github.com/software-mansion/cairo-profiler/tree/main).
+For the exact code implementation of this format, please refer to [this page](https://github.com/software-mansion/cairo-profiler/blob/main/crates/trace-data/src/lib.rs).
+For a practical example of how this format appears in a file, consider examining [this json file](./crates/cairo-coverage/tests/data/test_project/snfoundry_trace/tests::test_call::my_test.json).
```

Comment:
This is not really practical info 🤔 I think perhaps we should leave it for the end?

Reply:
Maybe mentioning that we in fact do plan to support `cairo-test`?